portfolio.html

```diff
@@ -8,6 +8,259 @@
     <link href="portfolio.css" rel="stylesheet">
 </head>
 <body>
```

Comment:
- [ ] As the previous reviewer mentioned your page is not centered. good job 👍🏼 implementing some of the changes however kindly make sure to center all the items on your page so it will reflect the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220827_185829](https://user-images.githubusercontent.com/25227022/187042680-66ec68a2-11ea-4fe2-b4c4-84b1c0cab0b6.png)|![](https://user-images.githubusercontent.com/86318284/187041314-871e3472-a825-4a5d-8436-9224c825a40e.png)|